scan/lib/scan/runner.rb

```diff
@@ -101,6 +93,26 @@ def handle_results(tests_exit_status)
       end
     end
 
+    def output_simulator_logs
+      return unless Scan.config[:include_simulator_logs]
+
+      UI.header("Collecting system logs")
+      Scan.devices.each do |device|
+        sim_resource_dir = FastlaneCore::CommandExecutor.execute(command: "xcrun simctl getenv #{device.udid} SIMULATOR_SHARED_RESOURCES_DIRECTORY 2>/dev/null", print_all: false, print_command: true)
+        logarchive_src = File.join(sim_resource_dir, "system_logs.logarchive")
+        FileUtils.rm_rf(logarchive_src) if File.exist?(logarchive_src)
```

Comment:
This is a bit redundant maybe. `rm_rf` ought to be enough as it handles the not `exist?`.